### PR TITLE
Better render code and pre in examples

### PR DESCRIPTION
**Pull request by Kiarokh**
fix: https://github.com/Lundalogik/crm-feature/issues/1334

## Review:
- [ ] Commits are [atomic](https://seesparkbox.com/foundry/atomic_commits_with_git)
- [ ] Commits have the correct *type* for the changes made
- [ ] Commits with *breaking changes* are marked as such

### Browsers tested:
(Check any that applies, it's ok to leave boxes unchecked if testing something didn't seem relevant.)

Windows:
- [ ] Chrome
- [ ] Edge
- [ ] Firefox

Linux:
- [ ] Chrome
- [ ] Firefox

macOS:
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

Mobile:
- [ ] Chrome on Android
- [ ] iOS

### CODE REVIEW

File: src/components/input-field/examples/input-field-textarea.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+@import '../../../style/internal/code-styles.scss';
+
+p {
+    font-size: small;
+}
+
+limel-input-field[type='textarea'] {
+    --textarea-height: 150px;
```
Comment (adrianschmidt) [inline src/components/input-field/examples/input-field-textarea.scss:8]: I think I disagree with this… 🤔

This means we're overriding the default look of the component in the example that's there to show what the component looks like. I'm not sure that makes a lot of sense.

Comment (adrianschmidt) [reply src/components/input-field/examples/input-field-textarea.scss:8]: Ah, this has just been moved…

Well, I still disagree with it being there in the first place then 😆

Comment (adrianschmidt) [reply src/components/input-field/examples/input-field-textarea.scss]: :zap:

**Review by jgroth — APPROVED** [review]
Looks great! :slightly_smiling_face:

**Review by jgroth — APPROVED** [review]

**Pull request merged**